templates/pages/paper.jsx

```diff
@@ -45,18 +45,19 @@ var Issue = React.createClass({
         return response.json();
     })
     .then((data) => {
-        this.setState({data: '', doi: ''});
+        this.setState({data: '', doi: '', 'submitted': true});
     });
     return;
   },
   getInitialState: function() {
-      return {data: '', doi: ''};
+      return {data: '', doi: '', 'submitted': false};
   },
   render: function() {
     return (
       <Page>
         <h1>Submit a Paper</h1>
         <p>This is a simple prototype demonstrating using a form to submit papers to Paper Badger, our contributorship badges prototype. In future versions we will integrate with publisher submission pipelines.</p>
+        <h4 hidden={!this.state.submitted}>You have succefully submitted the form</h4>
```

Comment:
Thank You :)
ᐧ

On Wed, Mar 16, 2016 at 1:35 AM, Abigail Cabunoc Mayes <
notifications@github.com> wrote:

> In templates/pages/paper.jsx
> <https://github.com/mozillascience/PaperBadger/pull/172#discussion_r56231009>
> :
>
> >    },
> >    render: function() {
> >      return (
> >        <Page>
> >          <h1>Submit a Paper</h1>
> >          <p>This is a simple prototype demonstrating using a form to submit papers to Paper Badger, our contributorship badges prototype. In future versions we will integrate with publisher submission pipelines.</p>
> > +        <h4 hidden={!this.state.submitted}>You have succefully submitted the form</h4>
>
> I like this, thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mozillascience/PaperBadger/pull/172/files/248d4e4ab866665976f611119f9a395302594fa8#r56231009>
>